explorer/nodewatch/nodewatch/NetworkRepository.py

```diff
@@ -107,6 +110,34 @@ def __init__(self, account_dict):
 		self.version = account_dict['version']
 
 
+class GeoLocationDescriptor:
```

Comment:
Is this class needed to filter the geo-location info?

Reply:
nope, this class is just an instance.

Reply:
Is the class needed since this should also work? Or am I missing something? 🤔 
``self.geo_location_map[entry['host']] = entry['geolocation']``

Reply:
yeah, it will look like this.

```py
if 'query' in entry['geolocation']:
    del entry['geolocation']['query']

self.geo_location_map[entry['host']] =  entry['geolocation']
```